### PR TITLE
Bracket spaces and double quotes added for lint check

### PR DESCRIPTION
**Pull request by joelyles**
### Motivation / Background

This change includes style updates to prevent lint check failure for those following the Getting Started with Rails Guide.

### Detail

This Pull Request updates code snippets for the article and comment models found in the Getting Started with Rails Guide - Chapter 9. Refactoring within the Using Concerns subsection. It will update the spacing within the array literal brackets and will change the single quotes to double quotes. These changes will be observed within the app/models/article.rb and app/models/comment.rb files for those following the Guide.

Without these changes, the lint job failed to pass after syncing to GitHub.

### Checklist

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`

### CODE REVIEW

**Comment by zzak** [comment]
Was the linter failing before this?

**Comment by joelyles** [comment]
> Was the linter failing before this?

Yes. The linter failed with out the modifications to the style. After I read through the annotations and updated the style. The linter passed. 

pr53321_01
pr53321_02

File: guides/source/getting_started.md
Comment (joelyles) [inline guides/source/getting_started.md:1]: I failed to disable my formatting extension before saving. More lines were changed than intended.

**Comment by zzak** [comment]
>  The linter failed without the modifications to the style

Where did the actual failures occur? Because I don't see them. Was this in your own fork?

**Comment by joelyles** [comment]
> > The linter failed without the modifications to the style
> 
> Where did the actual failures occur? Because I don't see them. Was this in your own fork?

The failures occurred in my own repo where I was following the Guide instructions. The repo is joelyles/getting-started-blog.

**Comment by zzak** [comment]
Ahh, ok sorry. I'm still getting over this virus so apologies for my dimness.

The problem here is that the code samples in the guide don't pass the omakase rubocop rules, those changes are usually accepted here. However, it would be preferable to rebase out those other changes from your editor that modified the markdown.

Thanks!

**Comment by joelyles** [comment]
> Ahh, ok sorry. I'm still getting over this virus so apologies for my dimness.
> 
> The problem here is that the code samples in the guide don't pass the omakase rubocop rules, those changes are usually accepted here. However, it would be preferable to rebase out those other changes from your editor that modified the markdown.
> 
> Thanks!

No worries, happy to clarify further! Rebase completed and PR updated to reflect final style modifications for the lint check.

File: guides/source/getting_started.md
Comment (joelyles) [reply guides/source/getting_started.md:1]: Completed rebase. File is now up to date.

**Comment by zzak** [comment]
@joelyles Sorry to be a pain, would you mind [squashing your commits](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

**Comment by joelyles** [comment]
@zzak No worries! Commits squashed. Let me know if there are any more issues here. Thanks!

**Comment by joelyles** [comment]
@zzak After pushing another rebase this PR closed. Apologies for this. Would it be best to open a new PR with the style changes?

**Comment by zzak** [comment]
@joelyles It's ok, I just merged it by hand in 7af392f227e2bf75e0c1504aa8eedd7b3c6102af. Thanks for your patch!

I think GitHub gets confused sometimes, sorry about that. :bow:

**Comment by joelyles** [comment]
> @joelyles It's ok, I just merged it by hand in [7af392f](https://github.com/rails/rails/commit/7af392f227e2bf75e0c1504aa8eedd7b3c6102af). Thanks for your patch!
> 
> I think GitHub gets confused sometimes, sorry about that. 🙇

@zzak I appreciate your help. Thanks!

**Pull request closed without merging**